nubis/puppet/init.pp

```diff
@@ -1,14 +0,0 @@
-import 'packages.pp'
-
-import 'apt.pp'
-import 'confd.pp'
-import 'consul.pp'
-import 'fluentd.pp'
-import 'jq.pp'
-import 'postfix.pp'
-import 'datadog.pp'
-import 'mig.pp'
-import 'credstash.pp'
-import 'consulate.pp'
-import 'nubis_lib.pp'
-import 'proxy.pp'
```

Comment:
These removals don't seem to have anything to do with this PR, other than that it looks good